IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/ElementHandler.cs

```diff
@@ -117,6 +117,6 @@ public bool WaitForEnabledState(IWebElement element, int secondsToTry = 5)
             }            
         }
 
-        private IWebDriver _Driver { get; set; }
+        private IWebDriver Driver { get; set; }
```

Comment:
It appears this is only set in the ctor and could be made readonly.

Reply:
Removed the setter.